app/controllers/application.js

```diff
@@ -1,6 +1,24 @@
 import Controller from '@ember/controller';
 import { inject as service } from '@ember/service';
+import { action } from '@ember/object'
 
 export default class ApplicationController extends Controller {
   @service router;
+  
+  @action openExplorerURL() {
```

Comment:
Nice code here to check sessionStorage and construct a url based on its values. 
An alternative you can explore is to move this code over to the route of the explorer page. 
This way, instead of defining this code at all the possible links to the Explorer page, you define it at the "gateway" of the Explorer page. So from whichever link or button the user clicks to get to the Explorer, this procedure to check sessionStorage still runs.

Reply:
Here is the Explorer route file where you might check session storage: https://github.com/NYCPlanning/labs-factfinder/blob/develop/app/routes/explorer.js

I assume the main point of the ticket is to restore all the data table configurations like source, topic filters, Compare To, show reliability etc. So as long as loading session state in the route doesn't override the user's selected geoid and geography, it should be ok